apollo-configservice/src/main/java/com/ctrip/framework/apollo/configservice/filter/ClientAuthenticationFilter.java

```diff
@@ -130,4 +155,11 @@ private boolean checkAuthorization(String authorization, List<String> availableS
     }
     return false;
   }
+
+  private void preCheckInvalidLogging(String message) {
+    logger.warn(message);
+    Tracer.logEvent("Apollo.AccessKey.PreCheck", message);
+    // only for test mock
+    accessKeyUtil.preCheckInvalid();
```

Comment:
We could change the method to `protected` and use `Spy` on the filter in the unit test to verify.

```java
clientAuthenticationFilter = spy(new ClientAuthenticationFilter(bizConfig, accessKeyUtil));
...
verify(clientAuthenticationFilter, never()).preCheckInvalidLogging(anyString());
```